Jarvis-Platform/jarvis-studio
# Hotfix/close icon position

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Fix close icon position while seeing item json in listing.


## Todos
- [x] Remove overflow
- [x] Add line-break


## Deploy Notes
Review & Merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout hotfix/close-icon-position
```

1. I'd like to have info on a workflow so I click on "DATA WORKFLOWS > Storage-to-Storage" in the left panel
2. I select a run or a configuration (see for ex. mirror-fd-io-exc-gdm-s-spl-in configuration)
3. I click on the eye icon to see the detailed description
4. Then I need to see something else so I'd like to close this current description
5. The close icon should be visible & accessible + no horizontal scroll


## Impacted Areas in Application
* All listing components
